### PR TITLE
Adds a Reassign Area proc to APCs, fixes template powernets even moreso

### PR DESCRIPTION
**Pull request by TDcoolguy300**
As above.

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
please merge

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -139,6 +139,11 @@
 		spawn(5)
 			src.update()
 
+/obj/machinery/power/apc/proc/reassign_area(turf/loc)
+	if(auto_name)
+		name = "[get_area(src)] APC"
+	area = src.loc.loc:master
```
Comment (Carbonhell) [inline code/modules/power/apc.dm:145]: Why colons,can't you do var/area/honk = get_area(src), area = honk.master

Comment (Carbonhell) [inline code/modules/power/apc.dm:145]: Also src reee

Comment (Carbonhell) [inline code/modules/power/apc.dm:145]: Nvm about src,this is the only case where using src is needed

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -139,6 +139,11 @@
 		spawn(5)
 			src.update()
 
+/obj/machinery/power/apc/proc/reassign_area(turf/loc)
```
Comment (Carbonhell) [inline code/modules/power/apc.dm:142]: Hol up what is turf/loc for,You're not using it

Comment (TDcoolguy300) [inline code/modules/power/apc.dm:142]: Incase you pass a turf in

**Comment by Carbonhell** [comment]
Dangerous but acceptable bugfix,merging

**Pull request merged**